### PR TITLE
My change

### PR DESCRIPTION
**Pull request by eric-h-st**
I made a test change for git purposes

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -16,6 +16,8 @@
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
     <!--
+     This is my change
```
Comment (EfratKlause) [inline public/index.html]: the line seems incomplete

File: public/index.html [hunk 2]
```diff
@@ -16,6 +16,8 @@
...
+
```
Comment (EfratKlause) [inline public/index.html]: remove the extra line

**Review by EfratKlause — CHANGES_REQUESTED** [review]

File: public/index.html (hunk 1 above)
Comment (eric-h-st) [reply public/index.html]: I fixed it

File: public/index.html (hunk 2 above)
Comment (eric-h-st) [reply public/index.html]: Removed

**Pull request merged**